analyses/integrative-analysis/01-integrative-analysis.Rmd

```diff
@@ -0,0 +1,366 @@
+---
+title: "Integrative analysis for sc-/sn-RNA-Seq Analysis in 10X Genomics data"
+author: "Antonia Chroni for SJCRH DNB_BINF_Core"
+author: "Antonia Chroni for SJCRH DNB_BINF_Core"
```

Comment:
author line is duplicated

Reply:
```suggestion
```

Reply:
Resolved here b5c1ce8017ebfec75b0df9dc348a7a2922970556